content/discrete-math/concepts/congruences/congruences.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'Congruences'
+Description: 'Congruences refer to the relationship between two integers a and b when they have the same remainder after
+division by positive integer m that is greater than 1.'
+Subjects:
+  - 'Computer Science'
+  - 'Discrete Math'
+Tags:
+  - 'Arithmetic'
+  - 'Math'
+  - 'Logical'
+  - 'Logic'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Congruences** refer to the relationship between two integers, _a_ and _b_, that have the same remainder after division
+by positive integer, _m_, that is greater than 1.
+
+## _a_ divides _b_
+
+If _a_ and _b_ are integers with a $$\neq$$ 0 then _a_ divides _b_ if an integer _c_, exists given that _b = ac_.
+
+Explanation:
+
+- When _a_ divides _b_, it is denoted as _a_ | _b_.
+- _a_ is termed as factor or divisor of _b_ while _b_ is termed as multiple of _a_.
+- If _a_ | _b_ then _b_ / _a_ is an integer (According to _a_ divides _b_).
+- If a does not divide b, it is denoted with a $$\nmid$$ b
```

Comment:
Here as well (see preview)

Reply:
I get it but that is standard notation to denote the non-divisibility. Replacing it with some other operator / character might make the statement vague.